docs/getting-started.md

```diff
@@ -30,9 +30,9 @@ First, here are the things you need before starting:
   * USB Ethernet (wired) dongle
   * Bluetooth
 
-  For Raspberry Pi you will need to use the wired Ethernet connection first.
-  With PiStorms the EV3 options will work too since you have a screen to configure
-  them on.
+  For RasberryPi you can use wired Ethernet connection first.
+  With PiStorms the EV3 options will work too since you have a built-in screen to configure them on.
+  With BrickPi you can also setup other connection from the console using external screen and keyboard.
```

Comment:
This is completely subjective, but generally I think of "setup" as a noun (e.g. "my setup") and "set up" as a verb form ("I set up my EV3"). I'd rather have it changed here to "set up".